### PR TITLE
Screen capture api

### PR DESCRIPTION
**Pull request by koustov**
# Description

Code changes to include the screen-capture #24 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [X] My code follows the [style guidelines](https://github.com/atapas/webapis-playground/blob/master/HOW-TO-ADD-DEMO.md) of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: web-apis/screen-capture/index.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+const isSupported = () => {
+  return navigator.clipboard &&
+    navigator.clipboard.writeText &&
```
Comment (atapas) [inline web-apis/screen-capture/index.js:3]: @koustov change this with the screen capture's logic.

Comment (koustov) [reply web-apis/screen-capture/index.js]: @atapas updated.

File: utils/data/demos.js [hunk 2]
```diff
@@ -165,6 +165,18 @@ const DEMOS = [
       twitter: 'tapasadhikary',
       email: 'tapas.adhikary@gmail.com',
     },
+  },
+  {
+    id: '_screen_capture_',
+    title: '💻 Screen Capture',
+    desc: 'The Screen Capture demo introduces additions to the existing Media Capture and Streams API to let the user select a screen to capture as a media stream',
+    path: 'screen-capture',
+    canIUseURL: 'https://caniuse.com/',
```
Comment (atapas) [inline utils/data/demos.js:174]: It should point to the screencapture API of can I use

**Review by atapas — CHANGES_REQUESTED** [review]
@koustov One more tiny change

File: utils/data/demos.js (hunk 2 above)
Comment (koustov) [reply utils/data/demos.js]: Done @atapas

**Review by atapas — APPROVED** [review]
@koustov Looks good. Approving.

How about another enhancement on top of it to download the screen capture a different format? Want to take that up?

**Comment by atapas** [comment]
@koustov 

Thanks a lot for taking the issue and making the app better. Please feel free to pick up more, contribute more. I wish you a great #Hactoberfest ahead. You will be added to the contribution list right after we merge this PR. Don't forget to share about your contribution publicly and provide your support with a star ⭐.

Great work 🔥 !!!!

**Comment by atapas** [comment]
@all-contributors please add @koustov  for Code

**Pull request merged**